.eslintrc.js

```diff
@@ -13,20 +13,10 @@ module.exports = {
         'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
```

Comment:
Should we maybe use someting like https://github.com/ioBroker/ioBroker.js-controller/blob/f54793e0236711426849de067a94a1dcf426fbfe/.eslintrc.json#L8 

https://github.com/ioBroker/ioBroker.js-controller/blob/f54793e0236711426849de067a94a1dcf426fbfe/package.json#L60
https://github.com/ioBroker/ioBroker.js-controller/blob/f54793e0236711426849de067a94a1dcf426fbfe/package.json#L62

Reply:
sure